spec/models/spree/order_updater_spec.rb

```diff
@@ -1,5 +1,10 @@
 RSpec.describe Spree::OrderUpdater do
   describe '#update' do
+    before do
+      # Prevent client from being instanciated in test environment.
+      allow(SuperGood::SolidusTaxjar).to receive(:api)
+    end
```

Comment:
@forkata we can just change the instantiation of the API to use a soft-fail with fetching the env variables rather than a hard fail! 

I think this is 

```rb
client = ::Taxjar::Client.new(
          api_key: ENV["TAXJAR_API_KEY")],
          api_url: ENV.fetch("TAXJAR_API_URL") { "https://api.taxjar.com" } # Sandbox URL: https://api.sandbox.taxjar.com
        )
```

If pass `nil` for the `api_key` to taxjar, this will still bomb hard down the line when the taxjar client actually tries to make a request, similar to the behavior if you provided an invalid API key, and I think this is appropriate because `nil` is an invalid API key. 

This will allow the specs to pass, as the API can be instantiated without erroring

Reply:
@Noah-Silvera This seems like a very reasonable solution here. The alternative path down which me and @senemsoy went to refactor this from being instantiated as a default parameter is a bit more work and definitely a breaking change! I am happy with this approach, unless someone can think of any downsides!

Reply:
We've updated to the new syntax and removed the stubbing.